test/test847.js

```diff
@@ -0,0 +1,72 @@
+// Test for nested subqueries - Issue: Nested SubQueries do not work as expected
+if (typeof exports === 'object') {
+	var assert = require('assert');
+	var alasql = require('..');
+}
+
+describe('Test 847 - Nested Subqueries', function () {
+	before(function () {
```

Comment:
Please merge this into a single call to alasql with a ` wrapped multi line SQL with statements seperated by ;